sunbird-platform-core/common-util/src/main/resources/externalresource.properties

```diff
@@ -180,4 +180,5 @@ sunbird_cert_service_base_url=
 sunbird_cert_download_uri=/v1/user/certs/download
 #{0} instancename , {1} toaccountemail or phone in mask , {2} from account email/phone in mask
 sunbird_account_merge_body=All your {0} usage details are merged into your account {1} . The account {2} has been deleted
-sunbird_account_merge_subject=Account merge successfully
\ No newline at end of file
+sunbird_account_merge_subject=Account merge successfully
+error_visualization_threshold=100
```

Comment:
error_visualization_threshold to sunbird_user_upload_error_visualization_threshold

Reply:
done